Victoradukwu/More-Recipes
# #155181719 user should be able to modify a recipe

#### What does this PR do?
- Adds the recipe edit functionality
#### Description of Task to be completed?
- modify the server side recipe controller to return recipes in descending order of  recipe id
- Use react and redux to implement delete function
#### How should this be manually tested?
- Start the app with the command `npm run start:dev`
- Go to localhost:8000 to see the landing page
- Navigate to `My Recipes page`
- Click on the name of a recipe. This takes you to a page where you can edit the recipe.
- Modify the recipe and submit. A message will be displayed showing the operation is a success
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
 - feature/155181719/user-should-be-able-to-modify-a-recipe

#### Screenshots (if appropriate)

#### Questions:
N/A
